src/Jobs_Templete/deployment.yaml.template

```diff
@@ -20,6 +20,11 @@ spec:
         userName: {{ job["user"] }}
         vcName: {{ job["vcName"] }}
         type: job
+      {% if "gpuType" in job %}
+        {% if job["gpuType"]|length > 0 %}
```

Comment:
Where is the definition of "length"?

Reply:
This is a jinja built-in https://jbmoelker.github.io/jinja-compat-tests/filters/length/